.github/workflows/cape-encrypt-release.yml

```diff
@@ -0,0 +1,85 @@
+name: Cape Encrypt Release
+
+on:
+  push:
+    tags:
+      - "[0-9]+.[0-9]+.[0-9]+"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-python@v4
+      with:
+        python-version: "3.10"
+    - name: Install pypa/build
+      run: pip install build --user
+    - name: Build wheel for cape_encrypt
+      run: python -m build --outdir dist cape_encrypt
+    - name: Install wheel & test installation
+      run: |
+        pip install $(find dist -name "*.whl")
+        python -c "import cape_encrypt; print(cape_encrypt.__version__)"
+    - name: Test wheel
+      run: |
+        pip install pytest absl-py
+        pytest cape_encrypt
+    - name: Upload wheels
+      uses: actions/upload-artifact@v2
+      with:
+        name: wheels
+        path: dist
+  
+  release-notes:
+    permissions:
+      contents: write
+      pull-requests: read
+    runs-on: ubuntu-latest
+    steps:
+    - name: Release drafter
+      id: release_drafter
+      uses: release-drafter/release-drafter@v5
+      with:
+        disable-autolabeler: true
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    outputs:
+      upload_url: ${{ steps.release_drafter.outputs.upload_url }}
```

Comment:
i'm not sure how this will mix w/ the release drafts of the other packages, but honestly i'm not too worried about it